src/rgw/rgw_op.cc

```diff
@@ -2896,7 +2896,6 @@ void RGWDeleteBucket::execute()
   }
 
   string prefix, delimiter;
-
```

Comment:
Plz do not introduce this kind of cleanup, if you're not touching around. This will pollute the commit history, which make us can not use `git show` and `git blame` to dig commit history.

Reply:
This also introduces the headache for resolving rebase conflict.

Reply:
Do you mean don't delete blank lines?

Reply:
Yes.